state_base.py

```diff
@@ -105,7 +110,7 @@ def _process_state(self, signal, group):
                 self.logger.exception(
                     "State Change failed for group {}".format(group))
                 return
-
```

Comment:
Change this from a public to protected variable: `changed_state` to `_changed_state`